am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/Access.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.amlib.enums.AccessType;
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+
+import java.util.Map;
+import java.util.Set;
+
+@Data
+@Builder
+public class Access {
+    private Map<JsonPointer, Set<Permission>> permissions;
+    private final AccessType accessType;
```

Comment:
This is not used. I see no reason to make that change now. I hate building something that brings no value.

I would be ready to take that out and address when we have story for that.